recipes/cdat_info/build.sh

```diff
@@ -0,0 +1,214 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
```

Comment:
I thought I pushed that change.  I am in a meeting in Seattle, I will be back to my office next week and I can make the change.